src/snowcli/cli/nativeapp/commands.py

```diff
@@ -1,8 +1,10 @@
 from typing import Optional
+from pathlib import Path
```

Comment:
As command was moved to `stage` subgroup, there should be no changes in this file. Added imports are now unused.